tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -291,4 +291,7 @@ set(test_src
     CommandLineInterface.unit.cc
     main.cc
 )
+# For CommandLineInterface.unit.cc, make an in.idf / in.epw in the test directory so we can test the legacy CLI (applies to CTest which sets the current directory to this one)
+configure_file("${PROJECT_SOURCE_DIR}/weather/USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw" "${CMAKE_CURRENT_BINARY_DIR}/in.epw" COPYONLY)
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/Resources/UnitaryHybridUnitTest_DOSA.idf" "${CMAKE_CURRENT_BINARY_DIR}/in.idf" COPYONLY)
```

Comment:
If you use ctest to run the tests, the in.epw / in.idf will already be there.

Reply:
:+1: